### PR TITLE
Refactor Mend CLI Folder

### PR DESCRIPTION
**Pull request by tabac-mend**
Performed the following:
1. Structure Mend CLI folder similar to UA
2. Added Readme to Mend CLI
3. Removed Conan from UA/GH Readme
4. Updated name convention from WS to Mend in GH Actions for UA

### CODE REVIEW

File: Mend CLI/README.md [hunk 1]
```diff
@@ -1,2 +1,56 @@
-# Mend CLI Examples by CI/CD Tool
-This folder contains tool specific examples of how to scan using the Mend Command Line Client within the CI/CD pipeline.
+# Examples by CI/CD Tool
+This repository contains tool specific examples of how to scan using the Mend [Unified CLI](https://docs.mend.io/bundle/cli/page/scan_with_mend_s_unified_cli.html) within a CI/CD pipeline.
+
+
+* [AzureDevOps](AzureDevOps)
+* [Bamboo](Bamboo)
+* [Bitbucket](Bitbucket)
+* [CodeFresh](CodeFresh)
```
Comment (lukebrogan-mend) [inline Mend CLI/README.md:8]: Remove this

**Review by lukebrogan-mend — CHANGES_REQUESTED** [review]

**Review by lukebrogan-mend — DISMISSED** [review]

**Comment by lukebrogan-mend** [comment]
A simpler unified CLI example can be found here, which we could use instead. Goal is to reduce number of files needed and complexity: https://github.com/lukebrogan-mend/Mend/blob/main/mend-azure-devops.yml

**Comment by samq-ws** [comment]
> A simpler unified CLI example can be found here, which we could use instead. Goal is to reduce number of files needed and complexity: https://github.com/lukebrogan-mend/Mend/blob/main/mend-azure-devops.yml

Looks great, Luke.  My only suggestion would be to add another comment beneath the trigger recommending to scan on push to main and push to release branches since scanning release branches and the main branch are a best practice for pipeline scans.  Then we should do this per technology in this refactor instead of having so many files.

Example below, feel free to reword
```
# Define your own pipeline trigger here:
# Below is defined a trigger for scanning on a push to the main branch and scanning on a push to release branches.  This is considered an ideal time to scan so that an SBOM can be created for software releases and inventory is managed for released software.
trigger:
- main
- release*
```

**Comment by tabac-mend** [comment]
The only downside with these examples is the need to "touch" the examples when having a call with a prospect. Therefore, created 4 examples per each use case. 

Let me know how to you want to proceed in order to update the PR.

**Comment by tabac-mend** [comment]
Updated the structure except for Github.com as we have a README with plenty of good examples for the pre-steps.
How do you want to implement it? if we do it in one file, the example for GH will be very long.

**Comment by samq-ws** [comment]
> Updated the structure except for Github.com as we have a README with plenty of good examples for the pre-steps. How do you want to implement it? if we do it in one file, the example for GH will be very long.

What README are you referring to?  The changes look really good so far.  Just trying to understand what you are referring to about GitHub.  In the future, please squash your commits before submitting a pull request as it is incredibly difficult to review right now.

**Comment by tabac-mend** [comment]
> > Updated the structure except for Github.com as we have a README with plenty of good examples for the pre-steps. How do you want to implement it? if we do it in one file, the example for GH will be very long.
> 
> What README are you referring to? The changes look really good so far. Just trying to understand what you are referring to about GitHub. In the future, please squash your commits before submitting a pull request as it is incredibly difficult to review right now.

This is the README (from UA space): https://github.com/mend-toolkit/mend-examples/blob/main/Unified%20Agent/CI-CD/GitHub/README.md

**Comment by samq-ws** [comment]
@tabac-mend , I'm going to merge this PR.  If you if you want to clean up the UA folder, please create a new PR.

**Review by samq-ws — APPROVED** [review]

**Pull request merged**